### PR TITLE
PLAT-259 Noop backend for ignoring stats completely

### PR DESCRIPTION
**Pull request by vgarvardt**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -59,6 +59,8 @@ func NewClient(dsn, prefix string) (Client, error) {
 		return NewLogClient(), nil
 	case "memory":
 		return NewMemoryClient(), nil
+	case "noop":
```
Comment (rafaeljesus) [inline client.go]: I find more elegant to create a const and use it instead of the strings:
```golang
	case StatsD:
	case Noop:
```

**Comment by rafaeljesus** [comment]
👍

File: client.go (hunk 1 above)
Comment (vgarvardt) [inline client.go]: fixed

**Comment by rafaeljesus** [comment]
:shipit:

**Pull request merged**